circle.yml

```diff
@@ -1,57 +0,0 @@
-# These environment variables must be set in CircleCI UI
-#
-# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
-# DOCKER_EMAIL   - login info for docker hub
-# DOCKER_USER
-# DOCKER_PASS
-#
-machine:
-  pre:
-    # Install CircleCI's fork for Docker 1.10.0
-    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
-    - chmod -R 777 $CIRCLE_TEST_REPORTS $CIRCLE_ARTIFACTS
-  node:
-    version: 8.2.0
-  services:
-    - docker
-  environment:
-    DEVELOPMENT: 1
-    CI_ENV: bash <(curl -s https://codecov.io/env)
-
-dependencies:
-  pre:
-    - docker info
-    # horrible set of hacks to install latest yarn (we need >= 1.2.1 for @neutrinojs/env)
-    - which yarn
-    - rm -f `which yarn`
-    - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-    - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-    - sudo apt-get update
-    - sudo apt-get install python-dev yarn
-  override:
-    # build Docker image first to make sure it'll build correctly
-    - ./bin/build
-
-test:
-  override:
-    # run tests using an upgraded version of docker-compose
-    - pip install --upgrade docker-compose
-    - cp .env-dist .env
-    # pull any missing images before running, so we can use
-    # --parallel (may not be needed once https://github.com/docker/compose/issues/4556
-    # fixed?)
-    - docker-compose pull --parallel
```

Comment:
It would be nice not to lose this, we added this explicitly to speed things up (unless there's something I'm missing and this is no longer required). See 29eb50e35c37dbfef4480d7cb91553e08f2c5eed

Reply:
My assumption was that this was fixed per https://github.com/docker/compose/pull/5787.

Reply:
Ok awesome :)

Reply:
Oh actually, maybe we'll need to upgrade the version of docker-compose we're using to get this change? Could you do that as a followup?

Reply:
https://github.com/mozilla/missioncontrol/blob/master/.circleci/config.yml#L50